DNN Platform/Components/Microsoft.AspNetWebApi/Microsoft.AspNetWebApi.dnn

```diff
@@ -30,6 +30,26 @@
               <name>System.Web.Http.WebHost.dll</name>
               <version>5.2.8</version>
             </assembly>
+            <assembly>
+              <path>bin</path>
+              <name>Newtonsoft.Json.Bson.dll</name>
+              <version>5.2.8</version>
+            </assembly>
+            <assembly>
+              <path>bin</path>
+              <name>Newtonsoft.Json.dll</name>

```

Comment:
Is it correct that we include this here, as it is already included in another package (https://github.com/dnnsoftware/Dnn.Platform/blob/ce45a3b2c4538101ba596076ac5bf24a7af51464/DNN%20Platform/Components/Newtonsoft/DotNetNuke.Newtonsoft.Json.dnn#L22)

I assume this is still ok as this is how third-party modules would behave, but I think this is the first time we have had this question come up.

Reply:
It won't hurt anything to include it twice, though I would recommend removing it since it does have its own package.

Reply:
I guess the right thing would be to add a dependency on the Newtonsoft *package*, rather than the DLL. But, if we're going to a quick fix here, what's in the PR doesn't break anything, so I'm fine leaving it as is.

Reply:
Hmm, good point on the package dependency, but if this package should try and install first, would it automatically reorder the package install order, anyone has experience with that scenario ? I would see a great benefit in using a package dependency as if the sub-dependency has sub-dependencies of its own, it solves that too.